js/app/modules/card/knowledgeDocument.js

```diff
@@ -237,7 +247,7 @@ const KnowledgeDocument = new Module.Class({
     _get_webview: function () {
         let webview = this._create_webview();
 
-        webview.renderer.enable_scroll_manager = this.show_toc;
+        webview.renderer.enable_scroll_manager = true;
```

Comment:
But the scroll manager isn't needed to do the scrolling, right? The `scroll_to()` method only does the fancy animation while updating the TOC. It _should_ be that if you navigate to `the.current/page#anchor` then WebKit will just move to that anchor as if you had navigated only to `#anchor`. In fact I'm wondering why that's not the case already in the code.